src/static/helpers/slasHelper.test.ts

```diff
@@ -340,23 +404,68 @@ describe('Registered B2C user flow', () => {
 });
 
 describe('Refresh Token', () => {
-  const expectedBody = {
-    body: {
-      client_id: 'client_id',
-      channel_id: 'site_id',
-      grant_type: 'refresh_token',
-      refresh_token: 'refresh_token',
-    },
-  };
-
-  test('refreshes the token', () => {
+  test('refreshes the token with slas public client', () => {
+    const expectedBody = {
+      body: {
+        client_id: 'client_id',
+        channel_id: 'site_id',
+        grant_type: 'refresh_token',
+        refresh_token: 'refresh_token',
+      },
+    };
     const token = slasHelper.refreshAccessToken(
       createMockSlasClient(),
       parameters
     );
     expect(getAccessTokenMock).toBeCalledWith(expectedBody);
     expect(token).toStrictEqual(expectedTokenResponse);
   });
+
+  test('refreshes the token with slas private client', () => {
+    const expectedReqOpts = {
+      headers: {
+        Authorization: `Basic ${stringToBase64(
+          `client_id:${credentialsPrivate.clientSecret}`
+        )}`,
+      },
+      body: {
+        grant_type: 'refresh_token',
+        refresh_token: parameters.refreshToken,
+      },
+    };
+    const token = slasHelper.refreshAccessToken(
+      createMockSlasClient(),
+      parameters,
+      {
+        clientSecret: credentialsPrivate.clientSecret,
+      }
+    );
+    expect(getAccessTokenMock).toBeCalledWith(expectedReqOpts);
+    expect(token).toStrictEqual(expectedTokenResponse);
+  });
+
+  test('refreshes the token warn users about using slas private client on', () => {
```

Comment:
This seems like a duplicate to the test above it, is the plan to modify this test to check the warning?

Reply:
good catch. It has been removed